src/commands/game_commands/music.ts

```diff
@@ -0,0 +1,182 @@
+import {
+    EMBED_SUCCESS_BONUS_COLOR,
+    getDebugLogHeader,
+    getUserVoiceChannel,
+    sendErrorMessage,
+    sendInfoMessage,
+    voicePermissionsCheck,
+} from "../../helpers/discord_utils";
+import { getGuildPreference } from "../../helpers/game_utils";
+import MusicSession from "../../structures/music_session";
+import MessageContext from "../../structures/message_context";
+import { state } from "../../kmq_worker";
+import BaseCommand, { CommandArgs, Help } from "../interfaces/base_command";
+import { IPCLogger } from "../../logger";
+import KmqMember from "../../structures/kmq_member";
+import { GameInfoMessage, GuildTextableMessage } from "../../types";
+import { chooseWeightedRandom } from "../../helpers/utils";
+import dbContext from "../../database_context";
+import Eris from "eris";
+import { KmqImages } from "../../constants";
+import { getTimeUntilRestart } from "../../helpers/management_utils";
+import Session from "../../structures/session";
+import GameSession from "../../structures/game_session";
+
+const logger = new IPCLogger("music");
+
+/**
+ * Sends the beginning of game session message
+ * @param textChannelName - The name of the text channel to send the message to
+ * @param voiceChannelName - The name of the voice channel to join
+ * @param message - The original message that triggered the command
+ * @param participants - The list of participants
+ */
+export async function sendBeginGameMessage(
+    textChannelName: string,
+    voiceChannelName: string,
+    message: GuildTextableMessage
+): Promise<void> {
+    const startTitle = state.localizer.translate(
+        message.guildID,
+        "command.play.gameStarting",
+        {
+            textChannelName,
+            voiceChannelName,
+        }
+    );
+
+    const gameInfoMessage: GameInfoMessage = chooseWeightedRandom(
+        await dbContext.kmq("game_messages")
+    );
+
+    const fields: Eris.EmbedField[] = [];
+    if (gameInfoMessage) {
```

Comment:
Will this ever be null?

Reply:
Similar behavior copied over from play.ts. Technically the table can be unpopulated, but I can remove it from both files